fabric-networking-v0/src/main/java/net/fabricmc/fabric/mixin/networking/MixinPlayerManager.java

```diff
@@ -38,10 +38,9 @@
 	public void onPlayerConnect(ClientConnection connection, ServerPlayerEntity player, CallbackInfo info) {
 		Optional<Packet<?>> optionalPacket = PacketRegistryImpl.createInitialRegisterPacket(ServerSidePacketRegistry.INSTANCE);
 
-		//noinspection OptionalIsPresent
-		if (optionalPacket.isPresent()) {
-			player.networkHandler.sendPacket(optionalPacket.get());
+		optionalPacket.ifPresent(packet -> {
```

Comment:
this is worse than the method ref above. this actually creates an anonymous inner class just because you like lambdas